subsys/nv_config/Kconfig

```diff
@@ -0,0 +1,85 @@
+#
+# Copyright (c) 2018 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig NV_CONFIG_SYSTEM
```

Comment:
can make all the NV (non volatile) to NVM (non volatile memory) ? NVM sounds more logical,

Just a suggestion :)